### ISSUE TITLE
New label, HP:0006527: lymphocyte-predominant inflammation

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#6439]
**HPO term**
Lymphoid interstitial pneumonia HP:0006527

**Preferred term label:**
lymphocyte-predominant inflammation 
(maybe better to differentiate between histological term and disease term?)

**Synonym to be added**
Lymphoid interstitial pneumonia

**Parent term (use hpo.jax.org/app)**
Interstitial pneumonitis HP:0006515

**broad/exact/narrow/plural/abbreviation ?**

**Your nano-attribution (ORCID)**
lmu: chrapp, @mgriese, @kknoflach

**Comment by pnrobinson** [comment]
@mgriese, @kknoflach  can we make a label that indicates where the lymphocyte-predominant inflammation is occurring?
What about

 Pulmonary interstitial lymphocyte-predominant infiltration

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4587010/

**Comment by pnrobinson** [comment]
Problably leave as separate new term

**Comment by pnrobinson** [comment]
Is this the same as Lymphocytic interstitial pneumonia (https://pubmed.ncbi.nlm.nih.gov/25359994/) and should it be added as a synonym?

**Comment by mgriese** [comment]
yes; the more frequent term is:
NEW TERM:	Lymphocytic interstitial pneumonia
Parent term:	Interstitial pneumonitis
Synonym:	Lymphocytic interstitial pneumonitis; Lymphoid interstitial pneumonitis/pneumonia
Definition:	
Lymphocytic interstitial pneumonitis is a benign lymphoproliferative disorders of the lung. It is characterized by the presence of a dense, predominantly lymphocytic interstitial infiltrate (lymphocytes, plasma cells, other elements of the lymphoreticular system) that expands the alveolar septa. 
Comment: 	
Lymphocytic interstitial pneumonitis is assumed to be a pathological immunologic response to different triggers and are particularly associated with autoimmune and infectious diseases as well as immunodeficiency.

Gene associations: COPA, CTLA4, LRBA, STAT3, TMEM173
PMID: 32040879

**Comment by pnrobinson** [comment]
added term
